lib/core/facets.rb

```diff
@@ -1,3 +1,10 @@
+unless $FACETS_LOADED
```

Comment:
Yeah, the naming is very lame here. I think your suggestion is good - will apply it. (I just did it in a slightly more "sloppy" fashion this time, given the anticipated short-lived nature of this fork.)